src/Content.tsx

```diff
@@ -7,13 +7,15 @@ import {
   ANSWER_CONTAINER,
   BUTTON_CONTAINER,
   CONTENT_CONTAINER,
-  DOWNLOAD_SVG,
   MAIN_CONTAINER,
   MODAL_CONTAINER,
   MENU_CONTAINER,
   CHAT_CONTAINER,
 } from "./utils/constants";
 
+import downloadImage from "./icons/download.png";
```

Comment:
Can we use a better download image please? this one does not look so good

<img width="79" alt="Screen Shot 2023-03-27 at 9 21 37 PM" src="https://user-images.githubusercontent.com/5238593/228102233-1ad9a488-13ed-4a5c-8c97-440bd69d8022.png">